app/models/project/life_cycle_step_definition.rb

```diff
@@ -42,15 +44,19 @@ class Project::LifeCycleStepDefinition < ApplicationRecord
 
```

Comment:
Only with the administration now existing it becomes apparent, that the user is not prevented from creating two steps with the same name.

Reply:
Added constraint